OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -81,6 +82,7 @@ public Minelayer(Actor self, MinelayerInfo info)
 		{
 			Info = info;
 			this.self = self;
+			ammoPools = self.TraitsImplementing<AmmoPool>().ToArray();
```

Comment:
as an optimisation we could send `ammoPools` to the LayMines activity

Reply:
Yea sounds good, done.